collections/ansible_collections/osac/service/plugins/filter/bcm.py

```
@@ -0,0 +1,224 @@
+    result = {}
+    for line in (notes or "").splitlines():
+        line = line.strip()
+        if "=" in line:
+            key, _, value = line.partition("=")
+            result[key.strip()] = value.strip()
+    return result
+
+
+def bcm_device_to_server(device: dict,
+                         racks_by_uuid: dict,
+                         disable_bmc_cert_verification: bool = True) -> dict:
```

Ruff (0.15.14)

[warning] Boolean-typed positional argument in function definition

(FBT001)